README.md

```diff
@@ -382,151 +424,454 @@ Jest uses Jasmine 2 by default. An introduction to Jasmine 2 can be found
+
+```js
+describe('drinking some La Croix', () => {
+  it('does not lead to errors', () => {
+    drinkSomeLaCroix();
+    expect(getErrors()).toBeFalsy();
+  });
+});
+```
+
+In JavaScript, there are six falsy values: `false`, `0`, `''`, `null`, `undefined`, and `NaN`. Everything else is truthy.
+
+### `.toBeGreaterThan(number)`
+
+To compare floating point numbers, you can use `toBeGreaterThan`. For example, if you want to test that `ouncesPerCan()` returns a value of more than 10 ounces, write:
+
+```js
+describe('ounces per can', () => {
+  it('is more than 10', () => {
+    expect(ouncesPerCan()).toBeGreaterThan(10);
+  });
+});
+```
+
+### `.toBeGreaterThanOrEqual(number)`
+
+To compare floating point numbers, you can use `toBeGreaterThanOrEqual`. For example, if you want to test that `ouncesPerCan()` returns a value of at least 12 ounces, write:
+
+```js
+describe('ounces per can', () => {
+  it('is at least 12', () => {
+    expect(ouncesPerCan()).toBeGreaterThanOrEqual(12);
+  });
+});
+```
+
+### `.toBeLessThan(number)`
+
+To compare floating point numbers, you can use `toBeLessThan`. For example, if you want to test that `ouncesPerCan()` returns a value of less than 20 ounces, write:
+
+```js
+describe('ounces per can', () => {
+  it('is less than 20', () => {
+    expect(ouncesPerCan()).toBeLessThan(10);
+  });
+});
+```
+
+### `.toBeLessThanOrEqual(number)`
+
+To compare floating point numbers, you can use `toBeLessThanOrEqual`. For example, if you want to test that `ouncesPerCan()` returns a value of at most 12 ounces, write:
+
+```js
+describe('ounces per can', () => {
+  it('is at most 12', () => {
+    expect(ouncesPerCan()).toBeLessThanOrEqual(12);
+  });
+});
+```
+
+### `.toBeNull()`
+
+`.toBeNull()` is the same as `.toBe(null)` but the error messages are a bit nicer. So use `.toBeNull()` when you want to check that something is null.
+
+```js
+function bloop() {
+  return null;
+}
+
+describe('bloop', () => {
+  it('returns null', () => {
+    expect(bloop()).toBeNull();
+  });
+})
+```
+
+### `.toBeTruthy()`
+
+Use `.toBeTruthy` when you don't care what a value is, you just want to ensure a value is true in a boolean context. For example, let's say you have some application code that looks like:
+
+```js
+drinkSomeLaCroix();
+if (thirstInfo()) {
+  drinkMoreLaCroix();
+}
+```
+
+You may not care what `thirstInfo` returns, specifically - it might return `true` or a complex object, and your code would still work. So if you just want to test that `thirstInfo` will be truthy after drinking some La Croix, you could write:
+
+```js
+describe('drinking some La Croix', () => {
+  it('leads to having thirst info', () => {
+    drinkSomeLaCroix();
+    expect(thirstInfo()).toBeTruthy();
+  });
+});
+```
+
+In JavaScript, there are six falsy values: `false`, `0`, `''`, `null`, `undefined`, and `NaN`. Everything else is truthy.
```

Comment:
I think it's fine if this sentence is the same - it's just trying to explain the difference between `falsy` and `truthy`, and the best way to understand that is to first learn what `falsy` is, regardless of which one is your end goal to understand.